doc/Dataplane Code Gen/DataplaneCodeGen_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,185 @@
+
+- Add back `Azure.Template.DataplaneCodeGen.Tests` csproj in your `Azure.<your-sdk-name>.sln`.
+- Rename the `Azure.Template.DataplaneCodeGen.Tests` project to `Azure.<your-sdk-name>.Tests`.
+- Rename the `TemplateServiceTestEnvironment.cs` file to `<client-name>TestEnvironment.cs` and update the test environment. Before running live tests you will need to create live test resources, please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/TestResources/README.md) to learn more about how to manager resources and update test environment.
+- Rename the `TemplateServiceLiveTests.cs` file to `<client-name>LiveTests.cs` and remove all the template project tests. Please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#live-testing) to add live tests.
+- Rename the `TemplateServiceTests.cs` file to `<client-name>Tests.cs` and remove all the template project tests and add all the unit tests in this file.
+- Remove the `SerializationHelpers.cs` file.
+
+**Note**: `Azure.<your-sdk-name>` is the Azure SDK package name and `<client-name>` is a client name, C# generator will generate a client which you can find in `Azure.<your-sdk-name>/Generated` directory.
+
+**Learn more:** see the [docs](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#to-test-1) to learn more about tests.
+
+## Samples
+
+In this section, we will talk about how to add samples. As you can see, we already have a `Samples` folder under `Azure.<your-sdk-name>/tests` directory. We run the samples as a part of tests. First, rename the `TemplateServiceSamples.HelloWorld.cs` file to `<client-name>Samples.HelloWorld.cs` and remove the existing sample tests. You will add the basic sample tests for your SDK in this file. Create more test files and add tests as per your scenarios.
+
+**Learn more:** For general information about samples, see the [Samples Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples).
+
+## Snippets
+
+Snippets are the great way to reuse the sample code. Please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets) to add snippets in your samples.
+
+## Export Public API
+
+If you make public API changes or additions, the `eng\scripts\Export-API.ps1` script has to be run to update public API listings. This generates a file in the library's directory similar to the example found in [sdk\template-dataplanecodegen\Azure.Template.DataplaneCodeGen\api\Azure.Template.DataplaneCodeGen.netstandard2.0.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-LLC/Azure.Template.LLC/api/Azure.Template.LLC.netstandard2.0.cs).
+
+Running the script for a project in `sdk\webpubsub` would look like this: 
+```
+eng\scripts\Export-API.ps1 webpubsub
+```
+
+## Readme
+
+README.md file instructions are listed in `Azure.<your-sdk-name>/README.md` file. Please add/update the README.md file as per your library.
```

Comment:
Could we link to the bare template in a **Learn More** section here?  Could we also link to a good example of a README for one of the Track 2 libraries, maybe Key Vault?